tests/console/gpg.pm

```diff
@@ -116,7 +111,15 @@ sub gpg_encrypt_file {
 }
 
 sub run {
-    select_console 'user-console';
+    select_console 'root-console';
+    # increase entropy for key generation for s390x on svirt backend
+    if (check_var('ARCH', 's390x') && (is_sle('>15') && (check_var('BACKEND', 'svirt')))) {
+        zypper_call('in haveged');
+        systemctl('start haveged');
+    }
+
+    # s390x on svirt backend needs root console to avoid corner case situation related to privileges
+    (check_var('ARCH', 's390x') && (check_var('BACKEND', 'svirt'))) ? select_console 'root-console' : select_console 'user-console';
```

Comment:
it must be done as user not root ?

Reply:
It must be done as a root on s390x svirt worker, otherwise it will hit specific situation.  The test was originally written for normal user and I would like to keep previous behavior for other archs.

Reply:
I mean you are selecting user console, I would not do it if it's not needed at all.

Reply:
Yes, the switching between consoles is not necessary and it can be done as root user for all archs.